README.md

```diff
@@ -2,15 +2,54 @@
 
 ## Description
 
-Jiminy is an open-source C++ simulator of poly-articulated systems, under the first restriction that the contact with the ground can be reduced to a dynamic set of points and the second restriction that the collisions between bodies or the environment can be neglected.
+Jiminy is a fast and lightweight open-source simulator for poly-articulated systems. It was built with two ideas in mind:
 
-It is built upon [Pinocchio](https://github.com/stack-of-tasks/pinocchio), which is an open-source implementing highly efficient Rigid Body Algorithms for poly-articulated systems. It is used to handle low-level physics calculations related to the system, while the effect of the environment on it is handled by Jiminy itself. The integration of time is based on the open-source library [Boost Odeint](https://github.com/boostorg/odeint).
+ - **provide a fast yet physically accurate simulator for robotics research.** Jiminy is built around
```

Comment:
Add somewhere something similarly to dart: "Provide comprehensive API for dynamic quantities and their derivatives, such as mass matrix, Coriolis force, gravitational force, other external and internal forces." : Precise that it is not only getter but also setters (through Pinocchio interface, it is only marginally extended by Jiminy itself)